packages/block-editor/src/components/index.js

```diff
@@ -12,6 +12,7 @@ export { default as BlockEdit } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
 export { default as BlockIcon } from './block-icon';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
+export { default as BlockNavigationList } from './block-navigation/list';
```

Comment:
How confident are we to expose this as a public API?

Reply:
@youknowriad It is difficult to see how this block navigator will evolve at the moment. 

I suppose a strategy could be to make a duplicate component in the block library for the navigation block, develop that, and then once it's a bit clearer look at moving those improvements back to the block-editor package.

Reply:
Maybe it's better to mark it "experimental" for the moment right?

Reply:
@youknowriad here's a pr for that https://github.com/WordPress/gutenberg/pull/17446

Reply:
Awesome thank you.